Add multiple policy info to disable WDAC policies (#5623)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/windows-defender-application-control/disable-windows-defender-application-control-policies.md

## Disable unsigned Windows Defender Application Control policies

> There may come a time when an administrator wants to disable a WDAC policy. For unsigned WDAC policies, this process is simple. Whatever method is being used to deploy the policy (ex. Group Policy) must first be disabled. Then simply delete the SIPolicy.p7b policy file from the following locations, and the WDAC policy will be disabled on the next computer restart:

Reviewer: @JohanFreelancer9 : There seems to be a logical gap in the first sentence containing changes. May I ask you to verify and advise?

Author: @illfated could you clarify what you mean? I'm open to wording changes, I'm just not seeing the gap you're referring to.

Reviewer: Sorry for the delay in replying to you. I think the edit made by JohanFreelancer9 below is closer to what I had in mind. The sentence
> Whatever method is being used to deploy the policy (ex. Group Policy) must first be disabled.

looks a little bit unfinished, or at least not clear enough to be immediately clear. I may write a comment to suggest further modification of JohanFreelancer9's suggestion , but his edit is basically closer to what I was thinking.

> There may come a time when an administrator wants to disable a WDAC policy. For unsigned WDAC policies, this process is simple. Whatever method is being used to deploy the policy (ex. Group Policy) must first be disabled. Then simply delete the SIPolicy.p7b policy file from the following locations, and the WDAC policy will be disabled on the next computer restart:

Reviewer 2: ```suggestion
There may come a time when an administrator wants to disable a WDAC policy. For unsigned WDAC policies, this process is simple. The method that is being used to deploy the policy (for example Group Policy) must first be disabled. Then simply delete the SIPolicy.p7b policy file from the following locations, and the WDAC policy will be disabled on the next computer restart:
```

Reviewer: Thank you for this suggestion, it is much closer to how I was thinking it should look. My idea is something like the following:
```suggestion
There may come a time when an administrator wants to disable a WDAC policy. For unsigned WDAC policies, this process is simple. The method used to deploy the policy (such as Group Policy) must first be disabled, then simply delete the SIPolicy.p7b policy file from the following locations, and the WDAC policy will be disabled on the next computer restart:
```
My suggestion might possibly hit a bit short of the mark you were aiming for, but it is just a suggestion to point out what I was thinking for its coherence.

-   &lt;EFI System Partition&gt;\\Microsoft\\Boot\\

-   &lt;OS Volume&gt;\\Windows\\System32\\CodeIntegrity\\

> Note that as of the Windows 10 May 2019 Update (1903), WDAC now allows multiple policies to be deployed to a device. To fully disable WDAC when multiple policies are in effect, you must first disable each method being used to deploy a policy. Then delete the {Policy GUID}.cip policy files found in the \CIPolicies\Active subfolder under each of the paths listed above in addition to any SIPolicy.p7b file found in the root directory.

Reviewer 2: ```suggestion
Note that as of the Windows 10 May 2019 Update (1903), WDAC allows multiple policies to be deployed to a device. To fully disable WDAC when multiple policies are in effect, you must first disable each method being used to deploy a policy. Then delete the {Policy GUID}.cip policy files found in the \CIPolicies\Active subfolder under each of the paths listed above in addition to any SIPolicy.p7b file found in the root directory.
```
